Dockerfile

```diff
@@ -72,16 +72,6 @@ RUN cd /usr/local/lvm2 \
 	&& make install_device-mapper
 # see https://git.fedorahosted.org/cgit/lvm2.git/tree/INSTALL
 
-# Install lxc
```

Comment:
ugh i know it's silly but if we could merge this at the same time as #17701 to suffer only one cache bust :)